src/input/actions.rs

```diff
@@ -827,15 +827,41 @@ impl State {
                         };
 
                         if let Some(direction) = dir {
-                            self.handle_shortcut_action(
-                                Action::SwitchOutput(direction),
-                                seat,
-                                serial,
-                                time,
-                                pattern,
-                                Some(direction),
-                                true,
-                            )
+                            match direction {
```

Comment:
This behaviour makes the most sense to me, we should go to previous or next workspace (depending on direction). Those actions also fall back to SwitchOutput if corresponding methods return `InvalidWorkspaceIndex`